vnext/Shared/OInstance.cpp

```diff
@@ -620,7 +622,9 @@ std::vector<std::unique_ptr<NativeModule>> InstanceImpl::GetDefaultNativeModules
       nativeQueue));
 
   // #10036 - Blob module not supported in UWP. Need to define property bag lifetime and onwership.
```

Comment:
this comment is no longer true? can remove the ifdef now?

Reply:
The comment is no longer accurate. I'll update, but the ifdef should still be there. Otherwise, we'll create multiple instances of the same module. Not a functional problem, but an unnecessary use of extra memory.